tests/unit/expire.tcl

```diff
@@ -297,4 +328,29 @@ start_server {tags {"expire"}} {
         set ttl [r ttl foo]
         assert {$ttl <= 98 && $ttl > 90}
     }
+
+    test {GETEX use of PERSIST option should remove TTL} {
+       r set foo bar EX 100
+       r getex foo PERSIST
+       r ttl foo
+    } {-1}
+
+    test {GETEX use of PERSIST option should remove TTL after loadaof} {
+       r set foo bar EX 100
+       r getex foo PERSIST
+       after 2000
+       r debug loadaof
+       r ttl foo
+    } {-1}
+
+    test {GETEX use of PERSIST option propagate as PERSIST command to replica} {
```

Comment:
```suggestion
    test {GETEX propagate as to replica as PERSIST, DEL, or nothing} {
```
i improved this test to cover DEL which wasn't covered by other tests, and to test no propagation,
maybe now the test you added in aof.tcl isn't needed anymore.

Reply:
Great catch I missed the `DEL` case thank you. I still retained the `aof.tcl` test as it validates there is no change in the file size.